### PR TITLE
docs: Add Bottlerocket example

### PR DESCRIPTION
**Pull request by Andrey9kin**
# PR o'clock

## Description

There were a couple of PRs requesting changes to the module to run AWS Bottlerocket based nodes. (for example #840)
I got Bottlerocket up and running without any changes to the module so wanted to document it and possibly prevent
other PRs from trying to introduce changes that could be easily avoided.

### Checklist

- [ ] CI tests are passing
- [ ] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

File: examples/bottlerocket/variables.tf [hunk 1]
```diff
@@ -0,0 +1,17 @@
+variable "k8s_version" {
+  description = "k8s cluster version"
+  default     = "1.18"
```
Comment (barryib) [inline examples/bottlerocket/variables.tf:3]: Can you please bump this to 1.20 ?

File: examples/bottlerocket/main.tf [hunk 2]
```diff
@@ -0,0 +1,64 @@
+terraform {
+  required_version = ">= 0.12.0"
```
Comment (barryib) [inline examples/bottlerocket/main.tf:2]: `required_version = ">= 0.13.1"` The root module requires TF 0.13

**Comment by barryib** [comment]
@Andrey9kin thanks for working on this. Can you please rebase your branch from master ?

**Comment by Andrey9kin** [comment]
@barryib rebased + fixed comments.. Thanks for looking into it!

**Review by barryib — APPROVED** [review]

**Comment by barryib** [comment]
Thanks @Andrey9kin for your contribution.

**Comment by Andrey9kin** [comment]
@barryib my pleasure. hope it will be useful for the community!

**Pull request merged**